tests/ludwig/config_validation/test_checks.py

```diff
@@ -233,3 +235,34 @@ def test_comparator_combiner_entities(entity_1: List[str], entity_2: List[str],
         config_obj = ModelConfig.from_dict(config)
         assert config_obj.combiner.entity_1 == ["a1"]
         assert config_obj.combiner.entity_2 == ["b1", "b2"]
+
+
+def test_check_concat_combiner_requirements():
+    config = yaml.safe_load(
+        """
+input_features:
+  - name: description
+    type: text
+    encoder:
+      type: embed
+      reduce_output: null
+    column: description
+  - name: required_experience
+    type: category
+    column: required_experience
+output_features:
+  - name: title
+    type: category
+combiner:
+    type: concat
+trainer:
+  train_steps: 2
+model_type: ecd
+"""
+    )
+
+    with pytest.raises(ConfigValidationError):
+        ModelConfig.from_dict(config)
+
+    config[COMBINER][TYPE] = "sequence_concat"
+    ModelConfig.from_dict(config)
```

Comment:
I feel like this adds a second dimension to this test which doesn't feel like it fits (although I get the purpose of it). Should we just break this up into two different tests that use the same config?

Reply:
This extra block confirms that Line 264 is indeed caused by the choice of the concat combiner (and not due to any other ConfigValidationError). I've added a comment to make this clearer.